zerver/lib/bugdown/__init__.py

```diff
@@ -133,9 +133,8 @@ def walk_tree(root: Element,
     return results
 
 # height is not actually used
-def add_a(root, url, link, title=None, desc=None,
-          class_attr="message_inline_image", data_id=None):
-    # type: (Element, Text, Text, Optional[Text], Optional[Text], Text, Optional[Text]) -> None
+def add_a(root: Element, url: Text, link: Text, title: Optional[Text]=None, desc: Optional[Text]=None,
```

Comment:
I feel like all multiline function signatures should have one argument per line. The `:`, `,` and `=` makes it very hard to distinguish between argument name and type when skimming though the code.

Reply:
But that would make it really big unnecessarily.

Reply:
So should I edit the commit and split that function signature onto multiple lines?